src/modules/Experiences/ExperiencesList/ExperiencesListItem.js

```diff
@@ -13,12 +13,12 @@ import { formatMonthYearDate, getWorkLength } from './helpers';
 const ExperiencesListItem = ({ experience, onUpdate, onDelete, classes }) => {
   const initial = experience.host ? experience.host[0] : '';
   const title =
-    `${experience.host}, ` +
-    (experience.type === 'Education'
+    experience.type === 'Education'
       ? `${experience.degree} in ${experience.title}`
-      : `${experience.title}`);
+      : `${experience.title}`;
   const showEndDate = experience.type !== 'Accomplishment';
   const showAchievements = experience.type !== 'Accomplishment';
+  const showWorkLength = experience.type !== 'Accomplishment';
```

Comment:
It's a little awkward that these settings are getting defined in two different places: here and in `configureForm` in the form code. Maybe we could combine those into a helper function somehow?

Reply:
done